test/types.ts

```diff
@@ -0,0 +1,201 @@
+export type Json = string | number | boolean | null | { [key: string]: Json } | Json[]
+
+export interface Database {
```

Comment:
Copied from postgrest-js. Didn't bring over the the whole database setup that postgrest-js has, since that seems like a lot for this change.